app/javascript/packs/searchParams.js

```diff
@@ -185,8 +185,10 @@ function search(query, filters, sortBy, sortDirection) {
     .then((response) => response.json())
     .then((content) => {
       const resultDivs = [];
+      const currentUser = userData();
```

Comment:
@aitchiss I used `userData()` here because it is used elsewhere in this same file.